python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -835,22 +836,30 @@ def _get_contents_hash(self, config_descriptor, entities):
         # the modtimes of all yml files found. When they're added to the digest,
         # we'll dump as json sorting alphanumeric on yml file name to ensure
         # consistent ordering of data.
+        scanned = 0
```

Comment:
we have a timer decorator - i suggest adding this to this method. And do a pass and see if we should add it somewhere else.